TODO.md

```diff
@@ -1,9 +1,3 @@
-* Update this code with the most recent code from kb_sdk.
-  * Before the update, the template data extraction code (e.g. the `forTemplates()` method
-    and dependencies) should be removed from the KIDL parser code. The code should be for parsing
-    and regenerating KIDL files, not handling templating.
-  * Similarly, the HTML generation code should be moved into the KIDL code, perhaps in
-    `kidlhtml` or `kidl.html`.
```

Comment:
Changed my mind here, if anything it should be moved to its own package, but for now leaving it in the SDK is fine